processor/k8sattributesprocessor/processor.go

```diff
@@ -227,7 +227,19 @@ func (kp *kubernetesprocessor) addContainerAttributes(attrs pcommon.Map, pod *ku
 			return
 		}
 	default:
-		return
+		switch {
```

Comment:
I think we can just add those two cases just parallel to the cases outside, no need to be nested into a default case, which is clearer?